.github/workflows/basic.yml

```diff
@@ -21,6 +17,10 @@ jobs:
         pool_tracking: ['ON', 'OFF']
         shared_library: ['OFF']
         os_provider: ['ON']
+    env:
+      BUILD_DIR : "${{github.workspace}}/build/"
+      INSTL_DIR : "${{github.workspace}}/build/install-dir"
```

Comment:
can we make this install-dir not a part of the git repo? Perhaps `${{github.workspace}}/../install-dir`?

Reply:
Sure, done.